www/havm10/SeminarPage02/js/main.js

```diff
@@ -0,0 +1,250 @@
+var startLongitude = $('#start-longitude');
+var startLatitude = $('#start-latitude');
+var endLongitude = $('#end-longitude');
+var endLatitude = $('#end-latitude');
+var map = $('#map');
+var length = $('#length');
+var time = $('#time');
+var ascent = $('#ascent');
+var descent = $('#descent');
+var save = $('#save');
+var nameInput = $('#name-input');
+var saveContents = $('#save-contents');
+
+var canvas = document.getElementById('canvas-draw');
```

Comment:
k cemu je tento canvas? melo by to byt jasne z nazvu promenne

Reply:
název upravený